### PR TITLE
fix/improve hook of dependency file form Cyclus repo

### PR DESCRIPTION
**Pull request by Baaaaam**
+note: the dependency part in `install_from_*.rst` file is a copy paste of the `CYCLUS_INSTALL.rst` file... I should include new flag and include that part too...

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -56,11 +56,11 @@ gh-clean gh-revert clean:
...
 gh-preview html:
-	wget -N https://raw.githubusercontent.com/cyclus/cyclus/develop/INSTALL.rst -O source/user/CYCLUS_INSTALL.rst
-	wget -N https://raw.githubusercontent.com/cyclus/cyclus/develop/DEPENDENCIES.rst -O source/user/CYCLUS_DEPENDENCIES.rst
-	sed -i 's/\`here\ \<DEPENDENCIES\.rst\>`\_/:doc:\`here\ \<CYCLUS_DEPENDENCIES\>\`/g' source/user/CYCLUS_INSTALL.rst
+	wget -N https://raw.githubusercontent.com/cyclus/cyclus/v1.4.0-release/INSTALL.rst -O source/user/CYCLUS_INSTALL.rst
+	wget -N https://raw.githubusercontent.com/cyclus/cyclus/v1.4.0-release/DEPENDENCIES.rst -O source/user/DEPENDENCIES.rst
+	sed -i s/\`here\ \<DEPENDENCIES\.rst\>\`\_/:doc:\`here\ \<DEPENDENCIES\>\`/g source/user/CYCLUS_INSTALL.rst
 	
-	wget -N https://raw.githubusercontent.com/cyclus/cycamore/develop/INSTALL.rst -O source/user/CYCAMORE_INSTALL.rst
+	wget -N https://raw.githubusercontent.com/cyclus/cycamore/v1.4.0-release/INSTALL.rst -O source/user/CYCAMORE_INSTALL.rst
```
Comment (scopatz) [inline Makefile]: I don't know if this is correct, since that branch should be removed once the release happens.

Comment (scopatz) [inline Makefile]: cool :)

**Comment by Baaaaam** [comment]
@scopatz this should fix it: I set a variable `FC_BRANCH` to set the branch to hook, and use `master` by default! 

thx for the review!

**Comment by Baaaaam** [comment]
this include also the change of the #211 fixing the flag name for the include start and end

**Comment by gonuke** [comment]
Thanks @Baaaaam

**Pull request merged**